### DISCUSSION TITLE
Request for Assistance with Accessing Complete 5G Metrics in WaveJudge Software

### DISCUSSION THREAD

**Discussion by Mohammed24699** [srsran/srsRAN_Project#867] [General Help]
Hello,

We have collected the IQ samples of srsRAN gNB and UE during their communication in various scenarios and we are using the WaveJudge software to analyze the 5G metrics. However, we are unable to see all the messages in the software. To access the complete set of messages, we need the values of block time index for the parameters PSS,PBCH and SSS

Any guidance or information you could provide on these values would be greatly appreciated.

Thank you for your assistance.

<img width="1280" alt="schedule" src="https://github.com/user-attachments/assets/3c0e3a9b-59a4-45cb-8686-852f43f089b1">

**Comment by fllay** [comment]
I follow Matlab tutorial using plutoSDR  to get IQ samples

https://www.mathworks.com/help/5g/ug/5g-nr-synchronization-signal-capture-using-software-defined-radio.html

then I follow this
https://www.mathworks.com/help/5g/ug/nr-cell-search-and-mib-and-sib1-recovery.html

I am not sure that it will give you all required information or not. 

BTW KeySight seems to be overly complicated to use. It requires some options on Fieldfox and software on PC. I saw a Aritsu demonstration, It seems a lot easier to use since  everything is on device itself.
